x-pack/filebeat/module/oracle/database_audit/_meta/fields.yml

```diff
@@ -0,0 +1,70 @@
+- name: database_audit
+  type: group
+  release: beta
+  description: >
+    Module for parsing Oracle Database audit logs
+  fields:
+    - name: status
+      type: keyword
+      description: >
+        Database Audit Status.
+
+    - name: sessionid
+      type: keyword
+      description: >
+        Session ID related to the audit event.
+
+    - name: client_terminal
+      type: keyword
+      description: >
+        If available, the client terminal type, for example "pty"
+
+    - name: client_address
+      type: keyword
+      description: >
+        The IP Address or Domain used by the client.
+
+    - name: client_user
+      type: keyword
+      description: >
+        The user running the client or connection to the database
+
+    - name: database_user
+      type: keyword
+      description: >
+        The database user used to authenticate.
+
+    - name: privilege
+      type: keyword
+      description: >
+        The privilege group related to the database user
+
+    - name: entryid
+      type: keyword
+      description: >
+        EntryID
+
+    - name: userhost
+      type: keyword
+      description: >
+        The host in which the database is running.
+
+    - name: action
+      type: keyword
+      description: >
+        The type of action logged in the audit event.
+
+    - name: action_number
+      type: keyword
+      description: >
+        Action number related to the audit event
+
+    - name: dbid
+      type: keyword
+      description: >
+        The ID of the DB/container accessed.
+
+    - name: length
+      type: long
+      description: >
+        The duration length.
```

Comment:
Can we extend this description and maybe give the field a more meaningful name? I like to think that Beats modules are built for experts and non-experts alike so if you are not an expert, by reading the description you can understand why this field is important in your Kibana dashboard. And if you are not going to write this into a Kibana dashboard maybe the field isn't that important 😄 I'm not an expert so I have no clue why this field is important 😂